### ISSUE TITLE
[Feature Request] Fast/Respawning/No Monsters Checkboxes

### ISSUE THREAD

**Issue by IrisWasTaken1** [lcferrum/qzdl#19]
I think it would be cool to have checkboxes which will add -fast, -nomonsters, and -respawn to the launch parameters.

**Comment by danicotra** [comment]
@RyanGlade : Yeah, I definitely agree with you ...and, indeed, I already made [a pull request for this](https://github.com/lcferrum/qzdl/pull/15) (Sadly, it was more than 2 years ago by now! Hope @lcferrum will attend to it some day...)

**Comment by spacebub** [comment]
@danicotra Hey, would you be willing to reimplement your commit [here](https://github.com/spacebub/qzdl) ? I forked the project in order to have a more modern and flexible build system with newer libraries and a few fixes/maintenance, since the original hasn't been active since 19'. There's also better build documentation in the COMPILE file.

**Comment by danicotra** [comment]
@spacebub 
Of course, yes, I'll be glad to do it :-D
I've the code ready, going with some pull requests just after I write this, thank you for your efforts in and your willing on keeping this project alive and get more modern 👍

**Comment by RyanGlade** [comment]
@spacebub 
Could you please enable the issues tab on your fork?

**Comment by spacebub** [comment]
@RyanGlade 
It should be open now

**Comment by RyanGlade** [comment]
@spacebub 
added a post to the issues tab 
https://github.com/spacebub/qzdl/issues/7

**Comment by RyanGlade** [comment]
‌

**Comment by RyanGlade** [comment]
‌
